Microsoft.Toolkit.Uwp.Notifications/Toasts/Compat/Desktop/InternalNotificationActivator.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if WIN32
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Windows.UI.Notifications;
+
+namespace Microsoft.Toolkit.Uwp.Notifications.Internal
+{
+    /// <summary>
+    /// Do not use this class. It must be public so that reflection can properly activate it, but consider it internal.
+    /// </summary>
+    public abstract class InternalNotificationActivator : InternalNotificationActivator.INotificationActivationCallback
```

Comment:
```suggestion
    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
    public abstract class InternalNotificationActivator : InternalNotificationActivator.INotificationActivationCallback
```

Reply:
Whoaaaa I didn't know you could do that! That's awesome!

@azchohfi do you think I should also use that to "remove" the DesktopNotificationManagerCompat classes too? I've obsoleted them, and there's essentially no reason anyone writing a new app should use those... I still left them there and functional so that existing apps can upgrade without breaking, but it seems like this disabling intellisense trick would be useful for keeping the coding path cleaner and more obvious (only ToastNotificationManagerCompat shows up in IntelliSense) while not breaking existing apps?

Reply:
We usually don't do that if we already Obsoleted something. You could, but I think it causes more frustration.
E.g.: Lets say you find a code sample, and:
1) You copy/paste it: It works. You get a warning. If you have warnings as errors, if fails to build. Very clear message on why.
2) You type the code: Intellisense doesn't work and you think VS (or the lib) is broken, but if you type **exactly** right, then it works.
I don't have any strong opinions on this, but I think Obsoleting them is enough.